crates/re_log_types/src/component_types/transform.rs

```diff
@@ -193,6 +193,13 @@ impl Pinhole {
 
 /// A transform between two spaces.
 ///
+/// The [`Transform`] component is special, as all instances of
+/// an Entity always share the same transform.
+/// In other words, a [`Transform`] component must always be a splat.
```

Comment:
Calling it a splat here is confusing I think... It's a mono-component, it'll be queried completely separately, never joined with anything else; it's in a realm of its own, not a splat :shrug:

Reply:
Well, if you have have an entity which is a point-cloud of ten points, and they all share the same transform, it _acts_ like a splat ("all these points have the same transform"), though not queried as such

Reply:
I improved the comment